contrib/pam_zfs_key/pam-config-zfs_key

```diff
@@ -0,0 +1,13 @@
+Name: Unlock zfs datasets for user
+Default: yes
+Priority: 128
+Auth-Type: Additional
+Auth:
+	optional	pam_zfs_key.so
+Session-Interactive-Only: yes
+Session-Type: Additional
+Session:
+	optional	pam_zfs_key.so
+Password-Type: Additional
+Password:
+	optional	pam_zfs_key.so
```

Comment:
`pam-config-zfs_key` and `pam_key` are identical.  Since `pam-config-zfs_key` isn't included in the Makefile.am I'm guessing this file was renamed but accidentally included in the diff.